template_src/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "my_app",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "none specified",
+  "license": "none specified"
```

Comment:
Cool, I didn't realize that `name` was automatically filled in.

I'd rather see the name be _My App_ or _MyApp_ but not _my_app_. Some may think it's nitpicking, but I'd like to see consistently across our project. Underscores are rarely used and I definitely wouldn't expect to see an app on my home screen with an underscore.

Version is cool to keep as 1.0.0.

The tests script is great to keep, since it encourages a best practice.

The keywords / author / license should either be obvious placeholders or just removed. I vote on removed entirely, since we don't have other npm elements.